org.wso2.codequalitymatrices/src/main/java/com/wso2/code/quality/metrics/Application.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.metrics;
+
+import org.apache.log4j.Logger;
+
+/**
+ * This is having the main method of this application
+ * PMT Access token, patch id and github access token
+ * should be passed in order as command line arguments when running the application.
+ *
+ * @since 1.0.0
+ */
+public class Application {
+    private static final Logger logger = Logger.getLogger(Application.class);
+
+    public static void main(String[] args) {
+        if (args.length == 3) {
+            String pmtToken = args[0];
+            String patchId = args[1];
+            String gitHubToken = args[2];
```

Comment:
IMHO, it might be better to offload tokens to a config file and only accept the patch ID as an argument, as token tend to be reused again and would be cumbersome to enter in the command line.